openvino2tensorflow/openvino2tensorflow.py

```diff
@@ -7272,6 +7275,7 @@ def representative_dataset_gen():
             print(f'{Color.REVERCE}Integer Quantization started{Color.RESET}', '=' * 56)
             converter = tf.lite.TFLiteConverter.from_saved_model(model_output_path)
             converter.experimental_new_quantizer = use_experimental_new_quantizer
+            converter._experimental_disable_per_channel = not use_per_channel
```

Comment:
Is there a reason why the same modification is added to 'saved_model_to_tflite.py' for 'output_integer_quant_tflite' but not to openvino2tensorflow.py? If there are no problems, it would be better to have the same fix for `output_full_integer_quant_tflite`.
https://github.com/PINTO0309/openvino2tensorflow/pull/114/files#diff-906bc49b3eeba462117c0cfaecd3c380a2309c9a20707ddcb200b4351d3eb63cR7296-R7297
![image](https://user-images.githubusercontent.com/33194443/179322730-3eb7a0b4-6846-4abc-9c4c-e820ca493d55.png)

Reply:
sorry, it was my mistake, I have added that in the corresponding line too.